docs/UserGuide.md

```diff
@@ -102,10 +104,16 @@ If you want a detailed look into each of the features that CampusCompanion has,
 ### Definitions 
 Here are some descriptions of the words we use throughout the User Guide:
 
+| Term                 | Definition                                                                                                                                                                                                          |
+|----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Command              | An input from the user that tells CampusCompanion to perform an action (eg. ADD an assignment, DELETE an assignment).                                                                                               |
+| Parameter            | Parameters are the details you would include about the assignment/internship (eg. Name, Description). A Command will include 1 or more parameters.                                                                  |
```

Comment:
`list-a` does not include any parameters if you're using it to just list assignments.
A Command **may** include 1 or more parameters.